recycler.sh

```diff
@@ -254,6 +254,26 @@ recreate_volume() {
   return 0
 }
 
+# Verifies that no fuse clients are connected to the volume given
+#
+# $1 - list of gluster endpoints
+# $2 - name of the volume
+volume_is_used() {
+  local rhost="${1%%,*}"  # extract the first entry
+  local vol_name="$2"
+  nfuse="$(
+    gluster --remote-host="$rhost" --mode=script \
+      volume status "$vol_name" client-list | \
```

Comment:
Does not work with `glusterfs 3.12.15`:

    No brick client-list in volume gluster-pv66

Reply:
We'll have to test what this means for 3.12 and lower clusters; depending on whether this is implemented in the client or the server we might be good...

Reply:
When used with pre-4.0 gluster cluster, the `client-list` command will return an empty list. This retains the current behavior of the recycler, but does not break it.